### PR TITLE
[ACS-4985] Added configuration to merge created and modified date filters into a single date filter

### PR DESCRIPTION
**Pull request by swapnil-verma-gl**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Created and Modified date filters are separate filters in the ACA UI

**What is the new behaviour?**
Added configuration to use the new SearchDateRangeAdvancedTabbed component, which allows merging the 2 date filters into a single UI Chip

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
For more info, visit [ACS-4985](https://alfresco.atlassian.net/browse/ACS-4985)

[ACS-4985]: https://alfresco.atlassian.net/browse/ACS-4985?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MichalKinas — COMMENTED** [review]
Please remember to review and fix e2es related to this ticket excluded in `e2e/protractor/protractor.excludes.json`. It can be done as part of another PR if you like but in this case please create a Jira for it

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "@angular/core": "14.1.3",
     "@angular/forms": "14.1.3",
     "@angular/material": "14.1.3",
+    "@angular/material-date-fns-adapter": "14.1.3",
     "@angular/material-moment-adapter": "14.1.3",
```
Comment (DenysVuika) [inline package.json:54]: do we need removing the moment one? or it's still needed?

Comment (swapnil-verma-gl) [reply package.json:54]: I'm not sure. I don't see usages of the moment adapter in ACA anymore, but there would still be its usages in ADF. There are usages of the base moment library in some ACA E2E test cases though

**Comment by swapnil-verma-gl** [comment]
> Please remember to review and fix e2es related to this ticket excluded in `e2e/protractor/protractor.excludes.json`. It can be done as part of another PR if you like but in this case please create a Jira for it

All the impacted e2e test cases from this change are in protractor. Since we are migrating over from protractor to playwright, I don't think we need to make any changes to them for now. I have added them to the protractor.excludes.json file for now.

The migration of these test cases can be tracked under [ACS-5007](https://alfresco.atlassian.net/browse/ACS-5007)

**Comment by nikita-web-ua** [comment]
I don't get why do we need `.env-original` ?

**Comment by swapnil-verma-gl** [comment]
> I don't get why do we need `.env-original` ?

We don't need that file. It probably got added by mistake when I was rebasing my branch to be up to date with develop. I've removed it now.

**Review by nikita-web-ua — APPROVED** [review]

File: e2e/protractor/protractor.excludes.json [hunk 2]
```diff
@@ -6,14 +6,27 @@
...
-  "C280051": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5632",
-  "C308042": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5632",
-  "C279186": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-4985",
-  "C279219": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-4985",
-  "C279221": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-4985",
-  "C279220": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-4985",
-  "C325006": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-4985",
+  "C279191": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
+  "C279192": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
+  "C279193": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
```
Comment (AleksanderSklorz) [inline e2e/protractor/protractor.excludes.json:11]: @swapnil-verma-gl why are you referring to https://alfresco.atlassian.net/browse/ACS-5183? That jira is in Verify state and I fixed already e2e which were failing as part of PR for that jira: 

https://github.com/Alfresco/alfresco-content-app/pull/3347/files

You should not mention that jira anymore. There are only two tests which were left and I was waiting for Michał and now I'm working on them, which are: 

"C280051": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5632",
"C308042": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5632"

And these two should be assigned to ACS-5632

Comment (swapnil-verma-gl) [reply e2e/protractor/protractor.excludes.json]: Updated the protractor.excludes.json file. This information was probably overwritten during rebase.

**Review by AleksanderSklorz — APPROVED** [review]

**Pull request merged**